packages/core/src/connectors/index.ts

```diff
@@ -81,7 +81,11 @@ export const loadConnectorFactories = async () => {
   return cachedConnectorFactories;
 };
 
-export const getLogtoConnectors = async (): Promise<LogtoConnector[]> => {
+export const getLogtoConnectors = async (options?: {
+  getConnectorStorage?: GetStorage;
```

Comment:
I am thinking that, the get/set handlers are only used in some functions, and they depends on each `request`, so pass it in "init" time is not a good way?